src/GeositeFramework/region.json

```diff
@@ -11,20 +11,22 @@
     },
     "headerLinks": [
         { "text": "Azavea", "url": "http://www.azavea.com/" },
-        { "text": "GIS", "url": "http://en.wikipedia.org/wiki/Geographic_information_system" }
-    ],
-    "regionLinks": [
```

Comment:
Seems like it may be less disruptive to keep the `regionLinks` key, so I'll update the framework to get its links from there & display them as pictured above.